src/features/bulk/bulk/selectors.ts

```diff
@@ -0,0 +1,44 @@
+import { createSelector } from '@reduxjs/toolkit';
+import { CustomFileLight } from '../../../types';
```

Comment:
It's called like this because some components return CustomFile from @amsterdam/bmi-component-library. The Light variant picks a few fields from that type.